### PR TITLE
Feat/user desk

### PR DESCRIPTION
**Pull request by NaamanTsur**
This PR adds the county display name to the app bar. Also I refactored the app bar code by removing the TopToolbar component and moving it to the AppToolbar.

### CODE REVIEW

File: client/src/components/App/AppToolbar/useAppToolbar.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        if (firstUserUpdate.current) {
+            firstUserUpdate.current = false;
+            getCountyByUser();
+        } else {
+            getUserActivityStatus();
+        }
+    }, [user]);
+
+    const getUserActivityStatus = () => {
+        axios.get(`/users/userActivityStatus`)
```
Comment (nevobonneh) [inline client/src/components/App/AppToolbar/useAppToolbar.tsx:39]: logs

File: client/src/components/App/AppToolbar/useAppToolbar.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                icon: 'error',
+                customClass: {
+                    title: classes.swalTitle
+                },
+            });
+        });
+    }
+
+    const setUserActivityStatus = (isActive: boolean) => {
+        axios.post('users/updateIsUserActive', {
```
Comment (nevobonneh) [inline client/src/components/App/AppToolbar/useAppToolbar.tsx:54]: logs

File: client/src/components/App/AppToolbar/useAppToolbar.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+                icon: 'error',
+                customClass: {
+                    title: classes.swalTitle
+                },
+            });
+        });
+    }
+
+    const getCountyByUser = () => {
+        axios.get('users/county/displayName').then((result) => {
```
Comment (nevobonneh) [inline client/src/components/App/AppToolbar/useAppToolbar.tsx:71]: logs

File: server/src/ClientToDBAPI/UsersRoute/mainRoute.ts [hunk 4]
```diff
@@ -55,4 +55,11 @@ usersRoute.get('/group', adminMiddleWare, (request: Request, response: Response)
         });
 });
 
+usersRoute.get('/county/displayName', adminMiddleWare, (request: Request, response: Response) => {
+    graphqlRequest(GET_COUNTY_DISPLAY_NAME_BY_USER, response.locals, { id: +response.locals.user.group })
```
Comment (nevobonneh) [inline server/src/ClientToDBAPI/UsersRoute/mainRoute.ts:59]: logs

**Review by nevobonneh — CHANGES_REQUESTED** [review]

**Review by nevobonneh — DISMISSED** [review]

**Review by nevobonneh — APPROVED** [review]

**Review by eladyashar — DISMISSED** [review]

**Review by eladyashar — APPROVED** [review]

**Review by daniel7764 — APPROVED** [review]

**Pull request merged**